util/opentitantool_test_runner.sh

```diff
@@ -44,6 +43,10 @@ for arg in "$@"; do
       FAIL="${arg#*=}"
       shift
       ;;
+    --test-bin=*)
```

Comment:
I think this is being changed to make the flash image sharable with the CW310, and that test-bin is a more generally applicable name for the binary artifact that's going to be loaded to the flash to perform a test. I personally prefer naming the binaries for where they should be placed, because I think that's what makes them unique. Should this be named something like --flash-image? or are we already committed to this name elsewhere?